src/locales/en/tdp.json

```
@@ -46,7 +46,8 @@
     "views": {
       "loading": "Loading...",
       "selectView": "Select a view",
-      "searchViews": "Search views"
+      "searchViews": "Search views",
+      "disabledTransition": "Please select 1 or more {{workbenchName}}(s) to jump to {{jumpTarget}}"
```

```suggestion
      "disabledTransition_one": "Please select one or more {{workbenchName}} to open the {{jumpTarget}} workbench",
      "disabledTransition_other": "Please select one or more {{workbenchName}}s to open the {{jumpTarget}} workbench"
```

Instead of writing `(s)` you can use the [i18next purals](https://www.i18next.com/translation-function/plurals) functionality.

Attention: You need to pass the `count` property in addition to the `workbenchName` when using the language string `disabledTransition`.

Please also be aware that just adding an `s` to the workbench name might not result in the correct plural (e.g., Solvents + s)

---

Ah! this is cool functionality and good to know. 

But I think the "(s)" is actually the more proper English? Because of its connection to "one or more"? For example in what situation, if I did it this way, would I actually not want an s on the end of the string? Other than if we already have one, of course. (Which we shouldnt? the entities should all be singular?) 

Very detoured now but just read this https://www.grammarphobia.com/blog/2015/04/parenthetical-plurals.html

---

Did make the other string changes though!

---

Thanks. That's fine with me. In the future we probably want to define the singular and plural of the entity name in the config.json. then we can use the correct form when necessary.